src/main/kotlin/Project2/Hello.kt

```diff
@@ -0,0 +1,5 @@
+package Project2

```

Comment:
Paketnamen bestehen üblicherweise aus Kleinbuchstaben. Muss hier, in der Testklasse sowie in den Ordnernamen geändert werden.